lib/faker/util.ex

```diff
@@ -22,6 +22,28 @@ defmodule Faker.Util do
     Enum.at(list, Faker.random_between(0, Enum.count(list) - 1))
   end
 
+  @doc """
+  Generate N unique elements
+
+  ## Examples
+
+      iex> Faker.Util.sample_unique(2, &Faker.Internet.email/0)
+      ["conor2058@schiller.com", "elizabeth2056@rolfson.net"]
```

Comment:
We're using 4 examples for doctests. Check other modules out. Thanks.

Reply:
Done!